crates/fluvio-hub-util/src/hubaccess.rs

```diff
@@ -50,12 +65,18 @@ impl HubAccess {
         }
     }
 
+    #[cfg(not(target_arch = "wasm32"))]
     pub fn default_load(remote: &Option<String>) -> Result<Self> {
         let cfgpath = default_cfg_path()?;
         let profileopt = std::env::var(FLUVIO_HUB_PROFILE_ENV).ok();
         HubAccess::load_path(cfgpath, profileopt, remote)
     }
 
+    #[cfg(target_arch = "wasm32")]
+    pub fn default_load(_remote: &Option<String>) -> Result<Self> {
+        unimplemented!("Support for `HubAccess::default_load` is not available for `wasm32`");
```

Comment:
should this panic, or  just return an error?

Reply:
I think returning error its fine as well!